frontend/packages/console-app/src/components/pdb/pdb-models.ts

```diff
@@ -33,8 +36,15 @@ export const pdbToK8sResource = (from: FormValues): PodDisruptionBudgetKind => {
       },
     },
   };
+
+  // Remove requiremet because only one of maxUnavailable and minAvailable in a single PodDisruptionBudget can be specify
+  const omitRequiremetFromExistingRes = _.omit(existingRes, [
```

Comment:
```suggestion
  // Remove requirement because only one of maxUnavailable and minAvailable in a single PodDisruptionBudget can be specify
  const omitRequirementFromExistingRes = _.omit(existingRes, [
```